server/app/views/v1/domain_authorizations/_domain_authorization.json.jbuilder

```diff
@@ -0,0 +1,9 @@
+json.id authorization.to_path
+json.domain authorization.domain
+json.challenge authorization.challenge
+json.challenge_opts authorization.challenge_opts
+json.authorization_type authorization.authorization_type
+if authorization.grid_service
```

Comment:
maybe always like this?
```
"linked_service": {
  "id": "<id>",
  "state": "<state>"
}
```

Reply:
Maybe `deploy_state`, as it refers only to the deployment state which was triggered to update the TLS_SNI cert? CLI really needs to only know when that deployment is finished.

Reply:
Right, `deploy_state` makes more sense.